common/app/Nav/nav.less

```diff
@@ -153,6 +153,28 @@ li.nav-avatar {
 }
 
 .night {
+  .nav-component-wrapper {
+    ::-webkit-input-placeholder {
+      color: @night-text-color;
+    }
+
+    ::-moz-placeholder {
+      color: @night-text-color;
+    }
+
+    ::-ms-placeholder {
+      color: @night-text-color;
+    }
+
+    ::placeholder {
+      color: @night-text-color;
+    }
+
+    .fcc_input {
+      background: @night-search-color;
```

Comment:
Here if you are adding just the color, then you should prefer longhand property for the background color.
If you set here a shorthand you will eventually set other background rules with their initial value. This is considered as an anti-pattern.

Reply:
@tomasvn can you explain to me what you mean by 'longhand property'? 

Thank you!

Reply:
@tomasvn nvm :) let me know if changes look good to you!